src/components/NewPlantForm.js

```diff
@@ -1,17 +1,58 @@
-import React from "react";
+import React, { useState } from "react"
+
+function NewPlantForm({ onHandleSubmitForm }) {
```

Comment:
Use handle or on, but not both. 
```suggestion
function NewPlantForm({ handleSubmitForm }) {
```